Products/CMFPlone/resources/browser/cook.py

```diff
@@ -107,7 +107,7 @@ def cookWhenChangingSettings(context, bundle=None):
                     logger.warn('Could not find resource: %s' , css_resource)
         if not resource.js or not js_path:
             continue
-        js_url = siteUrl + '/' + resource.js
+        js_url = resource.js
```

Comment:
what is the final url which is generated for js_url and css_url?
is it relative to the current context, e.g. when calling https://domain.com/subsite/context1/context2, is it https://domain.com/subsite/context1/context2/plone-logged-in.js or is it https://domain.com/subsite/plone-logged-in.js ?
If it's the first, that would be problematic as we cannot load the same resource for each context in the site separately.

Reply:
There is no final url for this resource (comment.js, event.js,..). 
js_url is used just for cooking. A somehow final url is the one of the cooked js plone-legacy-compiled.js